portfolio/src/main/webapp/script.js

```diff
@@ -86,7 +86,23 @@ function changePattern(pattern) {
     }
   }
 }
+function createCommentElement(eachComment) {
+    const divElement = document.createElement('div');
```

Comment:
I don't really see why you would need to make this a const here. var is used in a similar way in the below function. It's definitely fine, but would like to hear your thought on this.

Reply:
Most times, I use const for things I don't plan to change or update while the program is running. Basically, that's how I look at it. I don't know if that is right or not.

Reply:
You are mostly right. But I think you can think of this like this: divElement points to a variable, making it const will prevent the program from making it points to another variable. But you can still modify and update it by accessing its fields just like you did below.

You are still using it correctly though...just FYI:)

Reply:
Oh, okay. I'll take note of that.